docs/platform/release-notes/tizen-6-5-m1.md

```diff
@@ -0,0 +1,554 @@
+- capi-media-vision
+  - Barcode Detection
+    - Three attributes for images rotation have been added.
+      - MV_BARCODE_DETECT_ATTR_ROTATION_DEGREES
+      - MV_BARCODE_DETECT_ATTR_ROTATION_COUNT
+      - MV_BARCODE_DETECT_ATTR_ROTATION_DIRECTION
+    - Enumeration for the direction
+      - MV_BARCODE_DETECT_ATTR_ROTATION_CLOCKWISE
+      - MV_BARCODE_DETECT_ATTR_ROTATION_COUNTER_CLOCKWISE
+      - MV_BARCODE_DETECT_ATTR_ROTATION_ALL
+    - One attribute for image enhancement has been added.
+      - MV_BARCODE_DETECT_ATTR_USE_ENHANCEMENT
+- Media Content
+  - New API for getting media handle by path has been added.
+- Image Util
+  - Support for decoding HEIF (High Efficiency Image Format) has been added.
+    - Decode HEIC brand of HEIF
+
+#### Fixes
+
+- Muse server
+  - Deadlock problem of signal handler has been fixed by calling malloc of async-signal-safe internally.
+- Resource Manager
+  - A conflict issue caused by not released resource of the abnormal terminated client has been fixed.
+- Media Streamer
+  - Custom sink node freeze bug has been fixed.
+  - An unexpected error problem with stop() API has been fixed.
+
+
+### Network and Connectivity
+
+#### New and changed features
+
+- User Awareness API
+  - The API set for device has been added.
+  - The API set for user has been added.
+  - The API set for service has been added.
+  - The API set for monitor has been added.
+  - Following API set is required the Bluetooth or network privilege as the device type.
+    - Privilege : http://tizen.org/privilege/bluetooth or http://tizen.org/privilege/network.get
+    - Affected API set
+      - ua_monitor_add_sensor
+      - ua_monitor_remove_sensor
+      - ua_user_add_device
+      - ua_user_remove_device
+      - ua_device_update
+      - ua_monitor_start_scan
+      - ua_monitor_stop_scan
+      - ua_monitor_start_presence_detection
+      - ua_monitor_stop_presence_detection
+      - ua_monitor_start_absence_detection
+      - ua_monitor_stop_absence_detection
+  - Following API is required the location privilege.
+    - Privilege : http://tizen.org/privilege/location
+    - Affected API set
+      - ua_monitor_start_scan
+      - ua_monitor_stop_scan
+      - ua_monitor_start_presence_detection
+      - ua_monitor_stop_presence_detection
+      - ua_monitor_start_absence_detection
+      - ua_monitor_stop_absence_detection
+- Wi-Fi Manager API
+  - Wi-Fi connection logic has been optimized for better performance.
+- Wi-Fi
+  - MAC address randomization has been added.
+- Bluetooth Transport Discovery Service (TDS)
+  - An application scenario of enabling Wi-Fi Aware using Bluetooth Transport Discovery Service specification has been introduced.
+  - Devices, which want to communicate via Wi-Fi Aware technology, do not need to keep their Wi-Fi Aware interfaces to be activated always. Bluetooth TDS provides a feature of discovery available communication technologies between devices.

```

Comment:
```suggestion
  - Devices which can communicate Wusing i-Fi Aware technology, do not need to keep their Wi-Fi Aware interfaces to be activated always. Bluetooth TDS provides a feature of the discovery of available communication technologies between devices.
```
Please check for technical correctness

Reply:
will accept. I just modified errata."Wusing -> using Wi-Fi"
Devices which can communicate using Wi-Fi Aware technology, do not need to keep their Wi-Fi Aware interfaces to be activated always. Bluetooth TDS provides a feature of the discovery of available communication technologies between devices.

Reply:
Yes sorry for that.

Reply:
Thanks